docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,654 @@
+  [gem](https://en.wikipedia.org/wiki/RubyGems). You can install CocoaPods using the version of Ruby
+  that ships with the latest version of macOS.
+
+Learn more from the [Set up environment guide](/get-started/set-up-your-environment/).
+
+## Create an Expo project
+
+First, create an Expo project inside your existing native project's root directory.
+
+<Terminal cmd={['$ npx create-expo-app exp']} />
+
+This command creates a new directory named **exp** that contains your new Expo project. While you can name the project anything, this guide uses **exp** for consistency. The new project includes an example TypeScript application to help you get started.
+
+## Set up your project structure
+
+A standard React Native project places native code in **android** and **ios** directories. This structure may not be feasible when integrating with a large, existing native app.
+This guide covers two approaches: moving your native project files into the standard directories or setting up a monorepo.
+
+### Approach 1: Move your existing native project files
+
+Move your existing Android and iOS projects to the **android** or **ios** sub-directories, depending on the platform you are integrating with. For example:
+
+<Tabs>
+
+<Tab label="Android">
+
+<Terminal cmd={['$ mkdir exp/android', '$ mv /path/to/your/android-project exp/android/']} />
+
+</Tab>
+<Tab label="iOS">
+
+<Terminal cmd={['$ mkdir exp/ios', '$ mv /path/to/your/ios-project exp/ios/']} />
+
+</Tab>
+</Tabs>
+
+### Approach 2: Set up a monorepo
+
+Monorepos, or "monolithic repositories", are single repositories containing multiple apps or packages. [Learn more](/guides/monorepos/).
+
+Setting up a monorepo will ensure that Android and iOS scripts will be able to invoke commands from Node libraries even with a custom folder structure. To set up a Yarn monorepo, create a **package.json** file at the root of your project and add the following content:
+
+```json package.json
+{
+  "version": "1.0.0",
+  "private": true,
+  "workspaces": ["exp"]
+}
+```
+
+Then run `yarn install` to install the dependencies. This will ensure **node_modules** are installed at the root of your project, and that native scripts can interact with React Native code. Make sure to change `["exp"]` to the name of the Expo project you created in the previous step.
+
+> **info** Opting for a monorepo requires you to configure a custom project root, in Gradle/CocoaPods. This will be covered in the next sections.
+
+## Configuring your native project
+
+<Tabs tabs={["Configuring Android", "Configuring iOS"]}>
+<Tab>
+
+To integrate React Native on Android, you need to configure the native project by modifying the following files:
+
+- **Gradle files**: **settings.gradle**, top-level **build.gradle**, **app/build.gradle**, and **gradle.properties** to add the React Native Gradle Plugin (RNGP) and other properties.
+- **AndroidManifest.xml**: To add necessary permissions.
```

Comment:
it might be worth adding ([learn more](...)) to the permissions line and just linking to the heading on the section that discusses it below. my first reaction when seeing this is - what permissions and why? which are explained below, but i expect lots of people will wonder the same thing and want to have an explanation immediately available